operator/main.go

```diff
@@ -164,12 +163,12 @@ func main() {
 		setupLog.Info("/mutate-v1-pod webhook is registered")
 	}
 
-	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
-		setupLog.Error(err, "unable to set up health check")
+	if err := mgr.AddHealthzCheck("webhook", mgr.GetWebhookServer().StartedChecker()); err != nil {
```

Comment:
1. you could not change the checker's name which is referred to at https://github.com/apache/skywalking-swck/blob/master/operator/config/manager/manager.yaml#L58

2. The webhooks are active when `ENABLE_WEBHOOKS` is true. Based on that, you should move the webhook checkers to the above block with other webhooks, and enable `healthz.Ping` when `ENABLE_WEBHOOKS` is false.

Reply:
Thanks for correcting, updated it. BTW, could we merge the PR into 0.5.0 so that we can use the release version to add the java agent injector in the showcase, as the bug in the configmap controller will affect the injector in the showcase.

Reply:
all right, I will cancel the 0.5.0 vote.